driver/bpf/fillers.h

```diff
@@ -1929,6 +1929,13 @@ static __always_inline int bpf_accumulate_argv_or_env(struct filler_data *data,
 	*args_len = 0;
 	off = data->state->tail_ctx.curoff;
 
+	if(argv == NULL)
+	{
+		// we need to put a `\0` otherwise we could read junk data
+		data->buf[off & SCRATCH_SIZE_HALF] = '\0';
```

Comment:
this is why the CI was failing, if we called `execve` with `argv` or 'env` `NULL` we ended up reading random memory in `data->buf` written by the previous process on this CPU.